addons/binding/org.openhab.binding.icloud/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="icloud"
+						  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+						  xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+						  xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="bridge">
+		<label>iCloud Binding</label>
+		<description>Provides various information about an iCloud device.</description>
+
+		<channels>
+		    <channel id="Owner" typeId="Owner" />
+			<channel id="NumberOfDevices" typeId="NumberOfDevices"/>
+			<channel id="HttpStatusCode" typeId="HttpStatusCode" />

```

Comment:
Not sure if this channel make sense. If there is a communication problem to iCloud, better just put things to OFFLINE  state and put details to reason (e.g. HTTP status code).

Reply:
Thanks :-) ... removed the channel and log the status insted; the bridge details is set to communication error.

Reply:
You could also add status description to updateStatus method. 

Example:
updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "HTTP status = " + statusCode);

Description will be then shown in the UI's as well (not just on the logs).